src/psyclone/tests/psyir/transformations/extract_test.py

```diff
@@ -429,114 +379,92 @@ def test_node_list_dynamo0p3():
 
     schedule, _ = etrans.apply(schedule.children[0:3])
     code = str(psy.gen)
-    output = (
-        "      ! ExtractStart\n"
-        "      ! CALL write_extract_arguments(argument_list)\n"
-        "      !\n"
-        "      DO df=1,undf_any_space_1_f5\n"
-        "        f5_proxy%data(df) = 0.0\n"
-        "      END DO\n"
-        "      DO df=1,undf_any_space_1_f2\n"
-        "        f2_proxy%data(df) = 0.0\n"
-        "      END DO\n"
-        "      DO cell=1,f3_proxy%vspace%get_ncell()\n"
-        "        !\n"
-        "        CALL testkern_code_w2_only(nlayers, f3_proxy%data, "
-        "f2_proxy%data, ndf_w2, undf_w2, map_w2(:,cell))\n"
-        "      END DO\n"
-        "      !\n"
-        "      ! ExtractEnd\n")
+    # This output is affected by #637 (builtin support), and needs to be
```

Comment:
Again, please mark this as x-failing with a link to #637.

Reply:
Ditto - use pytest.xfail().